Source/FieldSolver/SpectralSolver/SpectralAlgorithms/GalileanPsatdAlgorithmRZ.H

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2019 David Grote
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_GALILEAN_PSATD_ALGORITHM_RZ_H_
+#define WARPX_GALILEAN_PSATD_ALGORITHM_RZ_H_
+
+#include "SpectralBaseAlgorithmRZ.H"
+
+/* \brief Class that updates the field in spectral space
+ * and stores the coefficients of the corresponding update equation.
+ */
+class GalileanPsatdAlgorithmRZ : public SpectralBaseAlgorithmRZ
+{
+
+    public:
+        GalileanPsatdAlgorithmRZ(SpectralKSpaceRZ const & spectral_kspace,
+                                 amrex::DistributionMapping const & dm,
+                                 int const n_rz_azimuthal_modes, int const norder_z,
+                                 bool const nodal,
+                                 const amrex::Array<amrex::Real,3>& v_galilean,
+                                 amrex::Real const dt_step);
+        // Redefine functions from base class
+        virtual void pushSpectralFields(SpectralFieldDataRZ & f) override final;
+        virtual int getRequiredNumberOfFields() const override final {
+            return SpectralFieldIndex::n_fields;
+        }
+
+        void InitializeSpectralCoefficients(SpectralFieldDataRZ const & f);
+
+        /**
+         * \brief Virtual function for current correction in Fourier space
+         * (equation (19) of https://doi.org/10.1016/j.jcp.2013.03.010).
```

Comment:
I think this reference was copied from the Cartesian docs and does not really point to the RZ equations:
```suggestion
         * \brief Virtual function for current correction in Fourier space.
```